src/kernel/outwards.jl

```diff
@@ -32,21 +32,90 @@ is occupied.
 
 Pass grid name `Symbol`s to `R` and `W` type parameters to use specific grids.
 """
-struct OutwardsDispersal{R,W,S<:Stencils.AbstractKernelStencil} <: SetNeighborhoodRule{R,W}
+struct _Mask end
+const NoMask = nothing
```

Comment:
I was confused by what NoMask was below. I would just use `nothing`, or make `NoMask` a real singleton like `_Mask`. 

You can even make `Mask` and `NoMask` singletons, then users could even use them directly. IDK